doc/developer_guide/coding-style.rst

```diff
@@ -156,7 +177,7 @@ Some important details:
                                (``:py:class:`psyclone.filename.Class```, 'int'
                                etc), or can also be a full sentence, then starting
                                with a capital letter and full punctuation.
-         exceptions            These must start with a lower case letter, and end
+         raises                These must start with a lower case letter, and end
```

Comment:
Iva has taken the ticket, but just my 2 cents: I prefer 'exceptions', since:

1. it is consistently using a noun on the left side in all rows of this table
2. the text reads better ('raises ... these must start'???)

if you prefer raises, imho the right side also need to be rewritten.

Reply:
I take your point. I wanted to use 'raises' as that's what it's actually called in the doc string but since I now link the two in the preceding paragraph I've decided to take the path of least resistance and put the entry in the table back to 'exceptions' :-)